community/kernel/src/main/java/org/neo4j/kernel/impl/newapi/DefaultRelationshipExplicitIndexCursor.java

```diff
@@ -52,6 +55,7 @@ public void initialize( ExplicitIndexProgressor progressor, int expectedSize )
     public boolean acceptEntity( long reference, float score )
     {
         this.relationship = reference;
+        read.singleRelationship( reference, scanCursor );
```

Comment:
Oh, so we don't keep the type, start-  and end-node information in the explicit relationship index - only the id?